FIPS/pom.xml

```diff
@@ -397,9 +397,10 @@
                        <pattern>com.google</pattern>
                        <shadedPattern>${shadeBase}.google</shadedPattern>
                      </relocation>
+                     <!-- google packages should be relocated explicitly to avoid problems with properties files renaming -->
                      <relocation>
-                       <pattern>google</pattern>
-                       <shadedPattern>${shadeBase}.google</shadedPattern>
+                       <pattern>google.geo</pattern>
```

Comment:
Why google packages are in such random locations? Can we gather them together?

Reply:
we can and I will fix it in a moment